coderd/database/awsiamrds/driver.go

```diff
@@ -0,0 +1,84 @@
+package awsiamrds
```

Comment:
Thoughts on naming these files `awsiamrds.go` along with the test file? Just follows other packages which generally do this.